protoc-gen-openapiv2/defs.bzl

```diff
@@ -68,6 +76,10 @@ def _run_proto_gen_openapi(
         extra_inputs.append(grpc_api_configuration)
         args.add("--openapiv2_opt", "grpc_api_configuration=%s" % grpc_api_configuration.path)
 
+    if grpc_api_configuration:
```

Comment:
I assume this should be `openapi_configuration`?

Reply:
Yes. FIXED.